backend/app/helpers/spree/admin/base_helper.rb

```diff
@@ -155,6 +155,11 @@ def spree_dom_id(record)
         dom_id(record, 'spree')
       end
 
+      def admin_layout(layout = nil)
+        @admin_layout = layout if layout
+        @admin_layout
+      end
+
```

Comment:
I think it's fine as is (but full disclosure, I may have suggested it). I don't like `@admin_layout = `, because we lose control over the assignment (juts in case we ever need to deprecate this or certain values). An `attr_writer` is fine, but reads a little awkward in a view. I don't think people normally think of the fact that the view context is an object when writing views..

``` ruby
<% self.admin_layout = 'foo' %>
```
vs my preferred
``` ruby
<% admin_layout 'foo' %>
```

I don't think `admin_layout`  being a setter and getter is out of place in a view, `content_for(name, content=nil)` is similar.

This is probably pretty subjective, so I'd love to hear others' thoughts.